packages/flutter/lib/src/painting/text_painter.dart

```diff
@@ -15,6 +15,19 @@ import 'text_span.dart';
 
 export 'package:flutter/services.dart' show TextRange, TextSelection;
 
+/// The different ways of considering the width of a Text widget.
```

Comment:
nit: use square brackets around any identifier (other than arguments that don't correspond to properties; those get backticks)

nit: this is in the painting layer, which doesn't have to be used with widgets. This can certainly talk about widgets, but it should also talk about TextPainter itself, which is what this layer knows about.